src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class RightTriangle {
```

Comment:
The class declaration should not be on the same line as the package declaration. Please move the class declaration to the next line for better readability and to adhere to Java conventions.